src/files-card-body.tsx

```diff
@@ -273,14 +274,16 @@ export const FilesCardBody = ({
             }
         };
 
-        const hasNoKeydownModifiers = (event: KeyboardEvent) => {
-            return !event.shiftKey && !event.ctrlKey && !event.altKey && !event.metaKey;
+        const hasNoKeydownModifiers = (event: KeyboardEvent, ctrlKey: boolean) => {
+            return !event.shiftKey && !ctrlKey && !event.altKey && !event.metaKey;
         };
 
         const onKeyboardNav = (e: KeyboardEvent) => {
+            const ctrlKey = isApple ? e.metaKey : e.ctrlKey;
```

Comment:
This added line is not executed by any test.

Reply:
Coverage complains as it checks both sides of the branch, so `isApple` is never true so no :100:  code coverage.